readme.md

```diff
@@ -116,6 +116,17 @@ final class PreferencesViewController: NSViewController {
 }
 ```
 
+## Localization
+
+This package supports localization but still in early development, so PRs are welcome!
+
```

Comment:
I think we should link to the `Localization` directory to show what localizations we currently support.